docs/addons/cloudwatch-adot-addon.md

```diff
@@ -0,0 +1,80 @@
+# AWS CloudWatch AWS Distro for OpenTelemetry (ADOT) Add-on
+
+[Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) collects monitoring and operational data in the form of logs, metrics, and events. You get a unified view of operational health and gain complete visibility of your AWS resources, applications, and services running on AWS and on-premises.  This add-on deploys an AWS Distro for OpenTelemetry (ADOT) Collector for CloudWatch which receives metrics and logs from the application and sends the same to CloudWatch console. You can change the mode to Daemonset, StatefulSet, and Sidecar depending on your deployment strategy.
+
+This driver is not automatically installed when you first create a cluster, it must be added to the cluster in order to setup CloudWatch for remote write metrics.
+
+For more information on the driver, please review the [user guide](https://docs.aws.amazon.com/eks/latest/userguide/opentelemetry.html).
+
+## Prerequisites
+- `cert-manager` Blueprints add-on.
+- `adot` EKS Blueprints add-on.
+
+## Usage
+
+This add-on can used with two different patterns :
+
+Pattern # 1 : Simple and Easy - Using all default property values. This pattern deploys an ADOT collector with `deployment` as the mode to write traces to CloudWatch console.
+
+```typescript
+import 'source-map-support/register';
```

Comment:
since we will be changing docs anyway, let's drop this source-map-support import. There is no need for it and it just occupies space and diverts attention from the add-on.

applies elsewhere.

Reply:
Fixed.